python/ray/serve/batching.py

```diff
@@ -304,6 +318,47 @@ def get_max_batch_size(self) -> int:
     def get_batch_wait_timeout_s(self) -> float:
         return self.batch_wait_timeout_s
 
+    def _get_curr_iteration_start_time(self) -> float:
+        """Gets current iteration's start time on default _BatchQueue implementation.
+
+        Returns -1 if the batch handler doesn't use a default _BatchQueue.
+        """
+
+        if hasattr(self.queue, "curr_iteration_start_time"):
+            return self.queue.curr_iteration_start_time
+        else:
+            return -1
+
+    async def _is_batching_task_alive(self) -> bool:
+        """Gets whether default _BatchQueue's background task is alive.
+
+        Returns False if the batch handler doesn't use a default _BatchQueue.
+        """
+
+        if hasattr(self.queue, "_handle_batch_task"):
+            if self.queue._handle_batch_task is None:
+                return False
+            else:
+                _, pending = await asyncio.wait(
+                    [self.queue._handle_batch_task], timeout=0
+                )
+                return len(pending) == 1
+        else:
+            return False
+
+    async def _get_batching_stack(self) -> str:
+        """Gets the stack for the default _BatchQueue's background task.
+
+        Returns empty string if the batch handler doesn't use a default _BatchQueue.
+        """
+
+        if hasattr(self.queue, "_handle_batch_task"):
+            str_buffer = io.StringIO()
+            self.queue._handle_batch_task.print_stack(file=str_buffer)
+            return str_buffer.getvalue()
+        else:
+            return ""
```

Comment:
better to return `None` to signal the task is not running

Reply:
I changed it to return `None`.